src/biogeochem/dynHarvestMod.F90

```diff
@@ -23,13 +23,15 @@ module dynHarvestMod
   use clm_varcon              , only : grlnd
   use ColumnType              , only : col                
   use PatchType               , only : patch                
+  use clm_varctl              , only : use_fates
   !
   ! !PUBLIC MEMBER FUNCTIONS:
   implicit none
   private
   !
   public :: dynHarvest_init    ! initialize data structures for harvest information
   public :: dynHarvest_interp  ! get harvest data for current time step, if needed
+  public :: dynHarvest_interp_resolve_harvesttypes  ! get harvest data for current time step, if needed, harvest-type-resolved
```

Comment:
Add a comment that this is only called for FATES, and CNHarvest is only called for use_cn and without FATES

Reply:
added comments on this.